NullPointerException: Cannot get property 'project' on null object when running gradle okbuck

When i run ``gradle okbuck --stacktrace``, i get this exception:

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':okbuck'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:84)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:55)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:61)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:58)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:88)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:45)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:233)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:215)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:74)
        at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:55)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:54)
        at org.gradle.internal.concurrent.StoppableExecutorImpl$1.run(StoppableExecutorImpl.java:40)
Caused by: java.lang.NullPointerException: Cannot get property 'project' on null object
        at com.uber.okbuck.core.model.base.Scope$_extractConfigurations_closure12.doCall(Scope.groovy:124)
        at com.uber.okbuck.core.model.base.Scope.extractConfigurations(Scope.groovy:109)
        at com.uber.okbuck.core.model.base.Scope.<init>(Scope.groovy:46)
        at com.uber.okbuck.core.model.base.Scope.<init>(Scope.groovy)
        at com.uber.okbuck.core.model.android.AndroidTarget.getMain(AndroidTarget.groovy:100)
        at com.uber.okbuck.core.model.base.Target.getProperty(Target.groovy)
        at com.uber.okbuck.composer.android.AndroidResourceRuleComposer.compose(AndroidResourceRuleComposer.groovy:17)
        at com.uber.okbuck.composer.android.AndroidResourceRuleComposer$compose.call(Unknown Source)
        at com.uber.okbuck.generator.BuckFileGenerator$_createRules_closure5.doCall(BuckFileGenerator.groovy:157)
        at com.uber.okbuck.generator.BuckFileGenerator.createRules(BuckFileGenerator.groovy:156)
        at com.uber.okbuck.generator.BuckFileGenerator.createRules(BuckFileGenerator.groovy:142)
        at com.uber.okbuck.generator.BuckFileGenerator$_createRules_closure1.doCall(BuckFileGenerator.groovy:89)
        at com.uber.okbuck.generator.BuckFileGenerator.createRules(BuckFileGenerator.groovy:77)
        at com.uber.okbuck.generator.BuckFileGenerator.generate(BuckFileGenerator.groovy:64)
        at com.uber.okbuck.generator.BuckFileGenerator$generate.call(Unknown Source)
        at com.uber.okbuck.OkBuckGradlePlugin$_apply_closure2$_closure7$_closure17.doCall(OkBuckGradlePlugin.groovy:104)
        at com.uber.okbuck.OkBuckGradlePlugin$_apply_closure2$_closure7.doCall(OkBuckGradlePlugin.groovy:103)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:588)
        at org.gradle.api.internal.AbstractTask$ClosureTaskAction.execute(AbstractTask.java:569)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:95)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:76)
```

```
→$gradle -v
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=1024m; support was removed in 8.0

------------------------------------------------------------
Gradle 3.2.1
------------------------------------------------------------

Build time:   2016-11-22 15:19:54 UTC
Revision:     83b485b914fd4f335ad0e66af9d14aad458d2cc5

Groovy:       2.4.7
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_91 (Oracle Corporation 25.91-b14)
OS:           Mac OS X 10.12.2 x86_64
```

--- a/buildSrc/src/main/groovy/com/uber/okbuck/core/model/base/Scope.groovy
+++ b/buildSrc/src/main/groovy/com/uber/okbuck/core/model/base/Scope.groovy
@@ -121,7 +121,10 @@ class Scope {
                 }
 
                 Target target = TargetCache.getTargetForOutput(targetProject, artifactFile)
-                if (target.project != project) {
+                if (target == null) {
+                    throw new IllegalStateException("No such artifact: ${artifactFile} for ${targetProject} with " +
+                            "artifact id: ${dependency}")
+                } else if (target.project != project) {
                     targetDeps.add(target)
                 }
             } else {

